CI/run_reframe.sh

```diff
@@ -93,9 +99,13 @@ git clone ${EESSI_CLONE_ARGS}
 export PYTHONPATH="${PYTHONPATH}":"${TEMPDIR}"/test-suite/
 
 # Start the EESSI environment
-unset MODULEPATH
-eessi_init_path="${EESSI_CVMFS_REPO}"/versions/"${EESSI_VERSION}"/init/bash
-source "${eessi_init_path}"
+if [ $UNSET_MODULEPATH == "True" ]; then
+    unset MODULEPATH
+fi
+if [ $USE_EESSI_SOFTWARE_STACK == "True" ]; then
+    eessi_init_path="${EESSI_CVMFS_REPO}"/versions/"${EESSI_VERSION}"/init/bash
+    source "${eessi_init_path}"
+fi
 
```

Comment:
Should we have an alternative to load a local module environment? Might not be needed on your systems. On our system we'd have to load a meta-module (e.g. `2023`, or `2024`) to make a software environment available.

Reply:
What could work is that we add something like `LOCAL_CI_MODULE_ENV=2024` and in our case `LOCAL_CI_MODULE_ENV=cpu_rome_256gb`. Now I'm also hardcoding that because the fix in ReFrame is not part of a release yet. I'll add that now now and than you can see if it works.